tests/libtest.sh

```diff
@@ -86,7 +81,7 @@ cleanup() {
   # cases it has been observed that udev rule kept the device
   # busy.
   udevadm settle
-  clean_config_files $INFILE $OUTFILE
+  rm -f $INFILE $OUTFILE
```

Comment:
This is library  and I feel it is better to pass in names of infile and outfile to clean instead of relying on global variables $INFILE and $OUTFILE.